.github/actions/setup/action.yml

```diff
@@ -19,3 +21,14 @@ runs:
     - name: Disable default go problem matcher
       run: echo "::remove-matcher owner=go::"
       shell: bash
+
+    #- name: Install cr-scale-operator
+    #  run: |
+    #    rm -rf cr-scale-operator
+    #    git clone https://github.com/test-network-function/cr-scale-operator
+    #    cd cr-scale-operator
+    #    IMG="quay.io/testnetworkfunction/cr-scale-operator"
+    #    make deploy IMG=$IMG
+    #    cd ..
+    #    rm -rf cr-scale-operator
+    #  shell: bash
```

Comment:
Can you remove the commented section?  Also the changes to the `description`?

Reply:
I think I have to add it back when you enable QE tests.

Reply:
What is description?

Reply:
Even if we uncommented this, the clusters at this point aren't spawned so the command to deploy would fail.

Reply:
We'll add this as part of the BeforeEach in the QE repo so it can run anywhere smoothly.